pandas/core/common.py

```diff
@@ -556,15 +556,6 @@ def require_length_match(data, index: Index) -> None:
         )
 
 
-# the ufuncs np.maximum.reduce and np.minimum.reduce default to axis=0,
-#  whereas np.min and np.max (which directly call obj.min and obj.max)
-#  default to axis=None.
-_builtin_table = {
-    builtins.sum: np.sum,
-    builtins.max: np.maximum.reduce,
-    builtins.min: np.minimum.reduce,
-}
-
 _cython_table = {
```

Comment:
Can `_cython_table` also ve removed?

Reply:
We need to know whether the passed function is a NumPy function (maybe there is a better way to do this?). This is because `Series.apply` treats these differently.